setup.py

```diff
@@ -370,7 +370,7 @@ def _get_package_path(package_name):
         break
     if arg == "-q" or arg == "--quiet":
         VERBOSE_SCRIPT = False
-    if arg in ["clean", "egg_info", "sdist"]:
+    if arg in ["clean", "dist_info", "egg_info", "sdist"]:
```

Comment:
slight nit:
This command is private and reserved for internal use of setuptools,
    users should rely on ``setuptools.build_meta`` APIs.
    
 Still though, looks harmless enough.

Reply:
@pytorchbot merge